OpenTESArena/CMakeLists.txt

```diff
@@ -78,6 +78,10 @@ IF (WIN32)
     ADD_DEFINITIONS("-D_SCL_SECURE_NO_WARNINGS=1")
 ENDIF()
 
+# Copy over required files
+FILE(COPY ${CMAKE_SOURCE_DIR}/data DESTINATION ${CMAKE_BINARY_DIR})
```

Comment:
The source directory refers to the top-level `CMakeLists.txt`, so it's already pointing to the right place.